src/themes.cpp

```diff
@@ -2370,7 +2371,7 @@ static void use_rose_gold_colors() {
 	Fl::background2(0xFF, 0xFF, 0xFF);
 	Fl::foreground(0x4C, 0x1E, 0x12);
 	Fl::set_color(FL_INACTIVE_COLOR, 0x60, 0x4C, 0x4C);
-	Fl::set_color(FL_SELECTION_COLOR, 0x15, 0x81, 0xFA);
+	Fl::set_color(FL_SELECTION_COLOR, 0x14, 0x79, 0xEA);

```

Comment:
These 4 slight color tweaks allow the contrast level of 50 to actually work (as opposed to needing to raise the contrast level even higher, which would have other negative effects).